src/SUMMARY.md

```diff
@@ -31,6 +31,7 @@
 
 # Servo’s architecture
 
+- [Crates under Servo\*](architecture/crates.md)
```

Comment:
No need for the * here; that's only for older content that hasn't been revised since importing it into the book.